python/django/security/path-traversal-join.yaml

```diff
@@ -0,0 +1,50 @@
+rules:
+  - id: path-traversal
+    patterns:
+      - pattern-inside: |
+          def $F(...):
+            ...
+      - pattern-not: |
+          ...
+          os.path.abspath(...)
+          ...
+      - pattern-not: |
+          ...
+          os.path.realpath(...)
+          ...
+      - pattern-either:
+        - pattern: |
+            $V = request.$W.get($X)
+            ...
+            $P = os.path.join(..., $V)
+            ...
+            open($P, ...)
+            ...
+            return django.http.HttpResponse(...)
```

Comment:
What happens if we remove this? This seems like it might limit the findings unnecessarily.

Reply:
Sure thing. Broader is fine with me!